src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -47,6 +47,8 @@ public enum ErrorCodeV1 {
   /** note: Only used by EmbeddingGateway */
   INVALID_REQUEST("Request not supported by the data store"),
 
+  INVALID_REQUEST_BAD_SYNTAX("Request invalid, JSON syntax incorrect"),
```

Comment:
Adding as old-style error code for minimal change right now, since code is called by `ThrowableToErrorMapper` with limited access.